quarklink-agent/scripts/ql_agent_install.sh

```diff
@@ -53,6 +53,16 @@ do
   echo "$line" >> $quarklink_config_dir/ql_ca_cert.pem
 done
 
+[ -f "$quarklink_config_dir/ql_sign_key.pem" ] && rm "$quarklink_config_dir/ql_sign_key.pem"
+#read signing key
+echo "Enter the signing key (you may have to press enter twice)"
+while read line
```

Comment:
In the above script we are using `while read -r line` but we are not here. Is that on purpose?

Reply:
Thanks